lib/services/delete_basket.rb

```diff
@@ -0,0 +1,31 @@
+Dir['./lib/**/*.rb'].each { |file| require file }
+
+module Shop
+  class DeleteBasket
```

Comment:
name can be better, code below do not delete basket (clean it) just remove one product from it